Sources/XCResultKit/Schema/ActionTestFailureSummary.swift

```diff
@@ -1,58 +1,61 @@
 //
-//  File.swift
-//  
+//  ActionTestFailureSummary.swift
+//
 //
 //  Created by David House on 7/3/19.
 //
 //- ActionTestFailureSummary
 //  * Kind: object
 //  * Properties:
-//    + message: String?
-//    + fileName: String
-//    + lineNumber: Int
-//    + isPerformanceFailure: Bool
-//    + uuid: String
-//    + issueType: String?
-//    + detailedDescription: String?
-//    + attachments: [ActionTestAttachment]
-//    + associatedError: TestAssociatedError?
-//    + sourceCodeContext: SourceCodeContext?
-//    + timestamp: Date?
-//    + isTopLevelFailure: Bool
-
+//	+ message: String?
+//	+ fileName: String
+//	+ lineNumber: Int
+//	+ isPerformanceFailure: Bool
+//	+ uuid: String
+//	+ issueType: String?
+//	+ detailedDescription: String?
+//	+ attachments: [ActionTestAttachment]
+//	+ associatedError: TestAssociatedError?
+//	+ sourceCodeContext: SourceCodeContext?
+//	+ timestamp: Date?
+//	+ isTopLevelFailure: Bool
+//	+ expression: TestExpression?
 import Foundation
 
 public struct ActionTestFailureSummary: XCResultObject {
```

Comment:
maybe it's a good opportunity to use `Sendable`, what di tou think ?

Reply:
I have not encountered any thread safety issues with the use of XCResultKit, which we've been using for a few years now rather heavily.   I did end up using it in a parallel multithreaded Task driven way to optimize performance, but as each Task really is just making an independent query into the underlying .xresult it has not caused problems.

Can you clarify why you think Sendable would be advantageous or useful here?

I'm not trying to make any more than the minimal changes to support Xcode 16.0 schema changes (beyond very minor cleanup along the way).